### PR TITLE
[Refactor] move APIFormat() of Issue and Label to convert package

### PR DESCRIPTION
**Pull request by 6543**
code refactor ...

### CODE REVIEW

**Comment by 6543** [comment]
@lunny can you have a look at this pleace?

**Comment by lunny** [comment]
CI is still failed.

**Comment by 6543** [comment]
@lunny It looks unrelated and tests pass localy - so :( ... wayt to hit the "update branch" buton ;)

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
@lunny CI passed 3 times now

File: modules/convert/issue.go [hunk 1]
```diff
@@ -5,10 +5,96 @@
...
 	"code.gitea.io/gitea/models"
 	api "code.gitea.io/gitea/modules/structs"
 )
 
+// ToAPIIssue converts an Issue to API format
+// it assumes some fields assigned with values:
+// Required - Poster, Labels,
+// Optional - Milestone, Assignee, PullRequest
+func ToAPIIssue(issue *models.Issue) *api.Issue {
+	if err := issue.LoadLabels(); err != nil {
```
Comment (guillep2k) [inline modules/convert/issue.go:19]: It seems that you could just call `issue.LoadAttributes()`. It also loads the assignees, but I don't think that's a problem.

File: modules/convert/issue.go [hunk 2]
```diff
@@ -36,3 +122,22 @@ func ToTrackedTimeList(tl models.TrackedTimeList) api.TrackedTimeList {
 	}
 	return result
 }
+
+// ToLabel converts Label to API format
+func ToLabel(label *models.Label) (apiT *api.Label) {
```
Comment (guillep2k) [inline modules/convert/issue.go]:
```suggestion
func ToLabel(label *models.Label) *api.Label {
```

File: modules/convert/issue.go [hunk 3]
```diff
@@ -36,3 +122,22 @@ func ToTrackedTimeList(tl models.TrackedTimeList) api.TrackedTimeList {
...
+	return &api.Label{
+		ID:          label.ID,
+		Name:        label.Name,
+		Color:       strings.TrimLeft(label.Color, "#"),
+		Description: label.Description,
+	}
+}
+
+// ToLabelList converts list of Label to API format
+func ToLabelList(labels []*models.Label) (apiT []*api.Label) {
```
Comment (guillep2k) [inline modules/convert/issue.go]:
```suggestion
func ToLabelList(labels []*models.Label) []*api.Label {
```

**Review by guillep2k — APPROVED** [review]

**Comment by techknowlogick** [comment]
ping @6543

File: modules/convert/issue.go (hunk 1 above)
Comment (6543) [reply modules/convert/issue.go:19]: I just moved this function, didn't want to change any behaviour

**Pull request merged**